### DISCUSSION TITLE
Customize the drawing toolbar

### DISCUSSION THREAD

**Discussion by JamesTheITManiac** [zsviczian/obsidian-excalidraw-plugin#2122] [Ideas]
It would be great if the drawing toolbar on the canvas can be customized to personal preference i.e. icons on the toolbar to be moved back and forth and the drawing tool cursor switchable between a crosshair /a pencil etc. 
![Screenshot 2024-11-20 195616](https://github.com/user-attachments/assets/63e9d13f-ebd1-4637-b053-ea100b38945d)

**Comment by LeonelRFF** [comment]
I think this should be requested at https://github.com/excalidraw/excalidraw/issues
